src/commands/beep.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   name: 'beep',
   description: 'Beep!',
-  execute(logger, message) {
+  execute (logger, message) {
+    // └[∵┌] └[ ∵ ]┘ [┐∵]┘
```

Comment:
That dance could probably be added to a new command! Like, make the bot reply with that! 😀